Formula/d/drogon.rb

```diff
@@ -0,0 +1,57 @@
+class Drogon < Formula
+  desc "Modern C++ web application framework"
+  homepage "https://drogon.org"
+  # pull from git tag to get submodules
+  url "https://github.com/drogonframework/drogon.git",
+      tag:      "v1.9.0",
+      revision: "f215cb15a0f53abd0ca7ee8b95ed8c9c3b40d262"
+  license "MIT"
+  head "https://github.com/drogonframework/drogon.git", branch: "master"
+
+  depends_on "brotli"
+  depends_on "c-ares"
+  depends_on "cmake"
+  depends_on "jsoncpp"
+  depends_on "ossp-uuid"
+
+  uses_from_macos "zlib"
+
+  def install
+    cmake_args = std_cmake_args
+    if OS.linux?
+      cmake_args << "-DUUID_LIBRARIES=uuid"
```

Comment:
Can these be detected if we depend on pkg-config?

Reply:
Possibly, let me try it.

Reply:
Seems not working. https://github.com/Homebrew/homebrew-core/actions/runs/6979802227/job/18993904524?pr=152693#step:4:157